torch/_dynamo/backends/onnxrt.py

```diff
@@ -1,131 +1,10 @@
-import importlib
-import logging
-import os
-import tempfile
-
-import torch
-from .common import device_from_inputs, fake_tensor_unsupported
+# This backend is maintained by ONNX team. To direct issues
+# to the right people, please tag related GitHub issues with `onnx`.
+#
+# Maintainers' Github IDs: wschin, thiagocrepaldi, BowenBao, abock
```

Comment:
Consider adding this file to CODEOWNERS with the github handles in this line.

PRs modifying this file auto adds code owners as reviewers.

Reply:
Ok. Also added myself to onnx files.